code/game/objects/items/handcuffs.dm

```diff
@@ -320,7 +320,7 @@
 	name = "bola"
 	desc = "A restraining device designed to be thrown at the target. Upon connecting with said target, it will wrap around their legs, making it difficult for them to move quickly."
 	icon_state = "bola"
-	breakouttime = 35//easy to apply, easy to break out of
+	breakouttime = 60//easy to apply, easy to break out of
```

Comment:
Doing this will make them have the same net effect, except now raiders have access to it since they can just make their own. Slowdowns are dangerous in combat.

Reply:
They should be- but knockdowns are far, far worse. Not being able to instantly remove it means that any conflict with a bola will likely become a melee fight.

Reply:
Changed to 4.5 seconds, feels good to me but we'll see in practice